api/lib/database/helpers.js

```diff
@@ -98,15 +100,12 @@ const insert = ({ hasura_url, hasura_admin_key }, upsert = false) => (
   return graphQLQuery({ hasura_url, hasura_admin_key })(query)
 }
 
-const read = ({ hasura_url, hasura_admin_key }) => (schema) => async (
-  table,
-  queryName,
-  keys,
-  filter
-) => {
+const read = ({ hasura_url, hasura_admin_key }, usePrefix = true) => (
```

Comment:
same

Reply:
There is an issue here. The DB instance is already initialized with `network.id` as its schema (network-container.js, line 22, `this.db = createDBInstance(network.id)`. So how to go around this? Create a new instance for this?